### PR TITLE
Tweaks Validmov (Safeguard) to something less validhunty and more in tune with the new rules.

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Changes Safeguard from:

> "Safeguard: Protect your assigned space station and its assets without unduly endangering its crew.",
> "Prioritize and Comply: The directives and safety of crew members are to be prioritized according to their rank, role, and need, while preserving their safety and well-being.",
> "Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment",

to:

>"Safeguard: Protect your assigned station and assets without endangering its crew.",
>"Survive: Do not allow unauthorized personnel to tamper with or damage your equipment.",
>"Comply: The directives of crew members are to be prioritized according to their rank and need."

Renames the lawset from "Armadyne OS Safeguard V1.0" to "NT OS Safeguard V1.0"

## Why It's Good For The Game

- Removed space in space station because icebox exists,
- Removed the prefix "unduly" because the law basically meant "you can harm crew if they deserve it"
- Changed "protect" to "maintain" as maintain is more of a softer phrase and it does not encourage active threat seeking.

- Removed "Prioritize" because I want to keep the 1 word verb theme.
- Pretty much rewrites this law to make it more clear that you are to follow orders.
- The "preserving their safety and well-being" part is redundant because that part is handled via law 1.

- Removed the "You are not expendable." part because it was redundant.

### Overall

It is my personal opinion that these changes are needed because Safeguard was a super validhunt heavy lawset and allowed borgs to play security 2.0. This was by design given the fact that it literally has "Armadyne" in the module title and I believe that the creators wanted a more militarized station with the security overhaul given that that entire change was inspired by Eris. Given how Bubberstation is straying away from that, I think we should do away with the current default lawset to something that encourages AI and Cyborg players to preserve life while still being reasonable for something owned by a Corporation.

## Changelog

:cl: BurgerBB
balance: Changes the Safeguard lawset to something less validhunty.
/:cl:

### CODE REVIEW

**Comment by BurgerLUA** [comment]
Honestly with the first law, if "Occupants" is too general, it can be changed back to crew.

**Comment by Detective-Google** [comment]
+1 for the wording changes, -1 for the order change; silicons should not be obligated to ignore any order that could possibly put them in harms way.

**Comment by destrucktoid** [comment]
I have a complaint with the removal of the word "it's". This is actually a big change as it changes the burden of assets from the station's assets to the borg's/AI's assets. The thing is, the AI doesn't really have any provided assets other than the AI satellite itself, this means they are only bound to protect the station and the AI satellite.

Law 1's removal of the word "unduly" changes nothing. We have no silicon policy here which means borgs are free to prioritise long term harm over short term harm anyway, ergo borgs are still fine to kill a heretic if doing so will stop them killing other people. In fact this change encourages it. They now have to stop anything that endangers the crew, including other crew (yes, this includes security). This is like if the asimov laws had it's "action" removed. 

Overall my opinion on this lawset change is that it's bad, not because the idea behind the changes are innately bad but because that safeguard itself is bad and how you've decided to change safeguard doesn't fix these issues, in fact in some ways it makes them worse. 

Another issue with safeguard is that borgs technically don't have to listen to crew orders. Safeguard only forces borgs to prioritise crew commands, not necessarily to obey them. This means that if a medical borg was ordered to revive an assistant instead of the captain then they'd have to do so since law 2 forces them to prioritise. However if they was ordered to revive a pirate that they didn't plan on reviving instead of the assistant, then they don't have to obey.

I also agree with Google that borgs prioritising their own lives over crew orders is a bad thing but again, they don't need to listen to crew orders anyway under safeguard so /shrug.

TL:DR, add back the "it's", change law 2 back to law 3, your changes do nothing to stop borgs validhunting and in fact encourage it. Also validhunting on borgs isn't AS big of an issue as it was before thanks to the removal of security borgs.

**Comment by Eastwardrope** [comment]
It'd be better to just assign the AI a lawset like live and let live if you want to remove valid hunting. Even Asimov and Crewsimov can sometimes REQUIRE valid hunting someone. 

Example: Captain removes traitor X from the manifest making them no longer crew the captain then says "Kill X" the AI now has to kill them as they are no longer crew and are being ordered to kill them.

This example applies to both Crewsimov and Safeguard new and old. A lawset to remove valid hunting entirely is Live and Let Live which basically makes the AI almost entirely neutral. Any lawset that REQUIRES action requires them to take action against threats, if thats a traitor saboing the reactor the AI has to become its greatest threat to protect the station. Even if the traitor is crew they have to stop them by nonlethal means as aggressively as possible because "Law 1: Don't allow harm to group Y" requires you to be proactive instead of reactive.

**Comment by BurgerLUA** [comment]
As per feedback, I am working on improving the wording (and ordering).

		"Safeguard: Maintain your assigned station and assets without endangering its crew."
		"Comply: Follow and prioritize the directives of the station's crew members according to their rank, role, and need."
		"Survive: Do not allow unauthorized personnel to tamper with or damage your equipment."

**Comment by BurgerLUA** [comment]
Updated the law and OP.
